test/jdk/javax/swing/border/Test4129681.java

```diff
@@ -44,7 +44,7 @@ public class Test4129681 {
 
     public static void main(String[] args) throws Exception {
         String testInstructions = """
-                When applet starts, you'll see a checkbox
+                When frame starts, you'll see a checkbox
```

Comment:
It's not what I really meant.

The instructions and the frame appear simultaneously, thus the tester doesn't need to wait for anything. Just describe what the tester needs to do.